File: C#/home/armstrong.cs
// armstrong

using System;

namespace Armstrong
{
    class Program
    {
        static void Main(string[] args)
        {
            int num, rem, sum = 0, temp;
            Console.Write("Enter the Number= ");
            num = int.Parse(Console.ReadLine());
            temp = num;
            while (num > 0)
            {
                rem = num % 10;
                sum = sum + (rem * rem * rem);
                num = num / 10;
            }
            if (temp == sum)
                Console.Write("Armstrong Number.");
            else
                Console.Write("Not Armstrong Number.");
        }
    }
}

File: Java/new java/Armstrong.java
public class Armstrong {

    public static void main(String[] args) {
        System.out.println(isArm(153));

    }

    static boolean isArm(int n) {
        int org=n;
        int sum = 0;
        while (n > 0) {
            int rem = n % 10;
            int cube = rem * rem* rem;
            sum = sum + cube;
            n /= 10;

        }
        if (sum == org) {
            return true;
        }

        return false;
    }

}
